pkg/controllers/vdb/obj_reconciler.go

```diff
@@ -241,7 +241,7 @@ func (o *ObjReconciler) checkForDeletedSubcluster(ctx context.Context) (ctrl.Res
 		// all pods in the subcluster.  If that isn't the case, we requeue to
 		// give those reconcilers a chance to do those actions.  Failure to do
```

Comment:
Can we update the comment? I would like to stop referring to admintools.conf. I think we can just remove the last sentence.